formule-demo/package.json

```diff
@@ -14,7 +15,8 @@
   },
   "dependencies": {
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-formule": "latest"
```

Comment:
this way we deploy latest publish version - we should either update to install formule from master, or run workflow on release